### ISSUE TITLE
Problem with LESS compilation

### ISSUE THREAD

**Issue by kReEsTaL** [bdkjones/CodeKit-1-Old#450]
In Codekit 1.3.1 (7839), compiling .less fils doesn't work like it used to. When I try to compile, I get this error message: 

"NameError: .container is undefined in /my/path/common.less:10:1
9 {
10  .container();
11  position: relative;"

.container is a mixin – never had any problem with it, it's something quite basic in LESS. If I remove the call to the .container mixin, Codekit breaks again on any other mixin I'm calling inside another class.

My mixin:

.container
{
    clear: both;
}

And the call:

.page
{
    .container;
}

Of course the mixin is written before the call.

I had to reinstall Codekit 6708 to make it work. 7834 and 7839 break.

**Comment by bdkjones** [comment]
I just tested the following Less code:

```
.container {
    color: white;
    margin-top: 5px;
}


.someOtherClass {
    .container;
}
```

It compiles, as expected, into the following CSS:

```
.container {
  color: white;
  margin-top: 5px;
}
.someOtherClass {
  color: white;
  margin-top: 5px;
}
```

Are you sure that your mixin class is exposed to the file where you're using it?

**Comment by bdkjones** [comment]
Haven't heard back and the test above passes, so I'm closing.

**Comment by kReEsTaL** [comment]
Hi!

It took us a while to do some tests to identify what was causing the compiling problem

It was this line:

`.span(@level:1)
{
   .column((@level * @column) + ((@level - 1) * @margin));
}`

Which we had to replace with:

`.span(@level:1)
{
   .column(@level * @column + (@level - 1) * @margin);
}`

Important note: this was happening with CodeKit 7839 because the Less version was updated to 1.3.1. We didn't have the problem with CodeKit 6708 whose Less version was 1.3.0.

Moreover, if we compiled our files with the command line along with Less 1.3.1, it _does_ compile. So my guess is there is a small bug in CodeKit which prevents the compiling to work.

Any idea? (Apart from not using brackets when there's no need to of course ^^')

Thanks in advance!

Marie
